lib/components/viewers/realtime-status-label.tsx

```diff
@@ -37,11 +40,11 @@ const TimeBlock = styled.div`
 
 const STATUS = {
   EARLY: {
-    color: '#337ab7',
+    color: blue[700],
     label: 'early'
   },
   LATE: {
-    color: '#D92923',
+    color: red[700],
```

Comment:
Why 700 over 800 that is used in the forms?

Reply:
In general I was trying to match the original colors to the new variables as closely as possible, but I agree this is a good opportunity to consolidate some of these